### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Saftladen
+
+[zaftˈlaːdn̩] Juice store; slightly disrespectful term used to describe a facility with bad organization or poor service
```
Comment (ipanova) [inline README.md]: s/disrespectful/impolite

File: README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+This repository is a bundle of scripts to generate asciinema demos for pulp topics without life user interaction.
```
Comment (ipanova) [inline README.md]: s/life/live

Comment (mdellweg) [reply README.md]:
```suggestion
This repository is a bundle of scripts to generate asciinema demos for pulp topics without live user interaction.
```

File: README.md (hunk 1 above)
Comment (quba42) [reply README.md]: I think "slightly disorganized" catches the meaning of how that term is most commonly used quite well.

Comment (ipanova) [reply README.md]: you guys know better ;) , i'd just rather avoid `disrespectful` word, imo

Comment (mdellweg) [reply README.md]: After consulting the dictionary another time, i think "disparaging" has the most fitting german translations. Nuances in language are so hard... 
```suggestion
[zaftˈlaːdn̩] Juice store; disparaging term used to describe a facility with bad organization or poor service
```

Comment (quba42) [reply README.md]: Rereading the full sentence, perhaps something like:
```
[zaftˈlaːdn̩] Juice store; a term often applied to arbitrary shops or establishments to imply that they are run in a somewhat disorganized or amateurish fashion.
```

Comment (mdellweg) [reply README.md]: Sorry, i was a little too fast here... Are you willing to open a PR? Also I am sure there is a lot that can be added to the README from experience once someone actually starts adding new casts.

**Pull request merged**